health/healthService_test.go

```diff
@@ -0,0 +1,27 @@
+package health
+
+import (
+	"testing"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/stretchr/testify/require"
+)
+
+func init() {
+	_ = logger.SetLogLevel("health:TRACE")
```

Comment:
do we need the health module trace logs in tests?

Reply:
Not needed, removed.